File: C# Basics/FirstStepsInCoding/ProjectCreation.cs
using System;

namespace ProjectCreation
{
    class Program
    {
        static void Main(string[] args)
        {
            string name = Console.ReadLine();
            int count = int.Parse(Console.ReadLine());
            int time = count * 3;
            Console.WriteLine($"The architect {name} will need {time} hours to complete {count} project/s.");
        }
    }
}

File: Java Basics/FirstStepsInCoding/ProjectCreation.java
package FirstStepsInCoding;

import java.util.Scanner;

public class ProjectCreation {
    public static void main(String[] args) {
        Scanner scan = new Scanner(System.in);

        String name = scan.nextLine();
        int projectsCount = Integer.parseInt(scan.nextLine());

        int hoursNeeded = projectsCount * 3;

        System.out.printf("The architect %s will need %d hours to complete %d project/s.", name, hoursNeeded, projectsCount);
    }
}
